setup.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: Setup Hush Line application server
+  hosts: all
+  become: yes
+  vars_files:
+    - vars/common.yml
+    - vars/secrets/secrets.yml
+  tasks:
+    - name: Install required packages
+      apt:
+        name:
+          - python3
+          - python3-pip
+          - git
+          - nginx
+          - default-mysql-server
+          - python3-venv
+          - certbot
+          - python3-certbot-nginx
+          - libnginx-mod-http-geoip
+          - ufw
+          - fail2ban
+          - redis
+          - redis-server
+          - apt-transport-https
+          - curl
+          - sqlite3
+          - acl
+        update_cache: yes
+        state: latest
+
+    - name: Clone Hush Line repository
+      git:
+        repo: https://github.com/scidsg/hushline.git
+        dest: /var/www/html/hushline
+        version: ansible
+
+    - name: Install Poetry
```

Comment:
This should be installed via `apt`

Reply:
I don't think poetry is available with `apt`. Their documentation suggests that the curl command is the "official" method: https://python-poetry.org/docs/#installing-with-the-official-installer